cmd/ttn-lw-cli/commands/config.go

```diff
@@ -71,30 +72,51 @@ func (c Config) getHosts() []string {
 	return getHosts(hosts...)
 }
 
-// DefaultConfig contains the default config for the ttn-lw-cli binary.
-var DefaultConfig = Config{
-	Base: conf.Base{
-		Log: conf.Log{
-			Level: log.InfoLevel,
+// BuildDefaultConfig builds the default config for the ttn-lw-cli binary for a given host.
+func BuildDefaultConfig(clusterHost string, insecure bool) Config {
+	grpcPort := map[bool]int{
+		true:  1884,
+		false: 8884,
+	}
+	httpPort := map[bool]int{
+		true:  1885,
+		false: 8885,
+	}
+	httpProto := map[bool]string{
+		true:  "http",
+		false: "https",
+	}
+	clusterGRPCAddress := fmt.Sprintf("%s:%d", clusterHost, grpcPort[insecure])
+	clusterHTTPAddress := fmt.Sprintf("%s://%s:%d", httpProto[insecure], clusterHost, httpPort[insecure])
```

Comment:
While I appreciate the creativity, please simply do

```suggestion
	clusterGRPCAddress := fmt.Sprintf("%s:8884", clusterHost)
	clusterHTTPAddress := fmt.Sprintf("https://%s:8885", clusterHost)
	if insecure {
		clusterGRPCAddress = fmt.Sprintf("%s:1884", clusterHost)
		clusterHTTPAddress = fmt.Sprintf("http://%s:1885", clusterHost)
	}
```